pkgs/by-name/di/dillo-plus/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fltk,
+  giflib,
+  libjpeg,
+  libpng,
+  openssl,
+  pkg-config,
+  wget,
+  xorg,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dillo-plus";
+  version = "3.3.0";
+
+  src = fetchFromGitHub {
+    owner = "crossbowerbt";
+    repo = "dillo-plus";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-NLerc1GXTdzuGVshXn7faK4vOu7wDVMiQNTljOF7OhA=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    fltk
+    giflib
+    libjpeg
+    libpng
+    openssl
+    xorg.libXdmcp
```

```suggestion
    libXdmcp
```

Since it is present on top-level